### DISCUSSION TITLE
How to install this package

### DISCUSSION THREAD

**Discussion by MaximeTut** [explosion/spaCy#11000] [Help: Installation] [answered]
Hi,
How can we install this via a Notebook ? 
**https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.0.0/en_core_web_sm-3.0.0.tar.gz**

Many thanks

**Comment by polm** [comment, marked as answer]
Please see the [install guide](https://spacy.io/usage). In a notebook you can run shell commands by starting them with `!`, so you should do something like this:

```
!python -m spacy download en_core_web_sm
```
